.kiro/specs/react-native-app/design.md

```
@@ -0,0 +1,615 @@
+    UI --> Query
+    Query --> Agg
+    Agg --> ESPN
+    Agg --> BTN
+    Agg --> Other
+    Store --> Storage
+    UI --> Store
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)